app/utils/print/IndividualProtectorPrintHelper.scala

```diff
@@ -0,0 +1,72 @@
+        bound.yesNoQuestion(NationalInsuranceNumberYesNoPage, "individualProtector.nationalInsuranceNumberYesNo", controllers.individual.routes.NationalInsuranceNumberYesNoController.onPageLoad(NormalMode).url),
+        bound.ninoQuestion(NationalInsuranceNumberPage, "individualProtector.nationalInsuranceNumber", controllers.individual.routes.NationalInsuranceNumberYesNoController.onPageLoad(NormalMode).url),
+        bound.yesNoQuestion(AddressYesNoPage, "individualProtector.addressYesNo", controllers.individual.routes.AddressYesNoController.onPageLoad(NormalMode).url),
+        bound.yesNoQuestion(LiveInTheUkYesNoPage, "individualProtector.liveInTheUkYesNo", controllers.individual.routes.LiveInTheUkYesNoController.onPageLoad(NormalMode).url),
+        bound.addressQuestion(UkAddressPage, "individualProtector.ukAddress", controllers.individual.routes.UkAddressController.onPageLoad(NormalMode).url),
+        bound.addressQuestion(NonUkAddressPage, "individualProtector.nonUkAddress",controllers.individual.routes.NonUkAddressController.onPageLoad(NormalMode).url),
+        bound.yesNoQuestion(PassportDetailsYesNoPage, "individualProtector.passportDetailsYesNo", controllers.individual.routes.PassportDetailsYesNoController.onPageLoad(NormalMode).url),
+        bound.passportDetailsQuestion(PassportDetailsPage, "individualProtector.passportDetails", controllers.individual.routes.PassportDetailsController.onPageLoad(NormalMode).url),
+        bound.yesNoQuestion(IdCardDetailsYesNoPage, "individualProtector.idCardDetailsYesNo", controllers.individual.routes.IdCardDetailsYesNoController.onPageLoad(NormalMode).url),
+        bound.idCardDetailsQuestion(IdCardDetailsPage, "individualProtector.idCardDetails", controllers.individual.routes.IdCardDetailsController.onPageLoad(NormalMode).url),
+        bound.dateQuestion(StartDatePage, "individualProtector.startDate", controllers.individual.routes.StartDateController.onPageLoad().url)
+      ).flatten
+
+    val amend: Seq[AnswerRow] = Seq(
+      bound.nameQuestion(NamePage, "individualProtector.name", controllers.individual.routes.NameController.onPageLoad(CheckMode).url),
+      bound.yesNoQuestion(DateOfBirthYesNoPage, "individualProtector.dateOfBirthYesNo", controllers.individual.routes.DateOfBirthYesNoController.onPageLoad(CheckMode).url),
+      bound.dateQuestion(DateOfBirthPage, "individualProtector.dateOfBirth", controllers.individual.routes.DateOfBirthController.onPageLoad(CheckMode).url),
+      bound.yesNoQuestion(NationalInsuranceNumberYesNoPage, "individualProtector.nationalInsuranceNumberYesNo", controllers.individual.routes.NationalInsuranceNumberYesNoController.onPageLoad(CheckMode).url),
+      bound.ninoQuestion(NationalInsuranceNumberPage, "individualProtector.nationalInsuranceNumber", controllers.individual.routes.NationalInsuranceNumberYesNoController.onPageLoad(CheckMode).url),
```

Comment:
Should be redirecting to NationalInsuranceNumberController

Reply:
Fixed now